src/graphdb.js

```diff
@@ -193,34 +257,73 @@ function graphDBInferenceSchema (graphbSchema, addMutations) {
     // query
     r += `type Query {\n`;
     gdbs.nodeStructures.forEach(node => {
-        let nodeCase = toPascalCase(node.label); 
-        r += `\tgetNode${nodeCase}(filter: ${nodeCase}Input): ${nodeCase}\n`;
-        r += `\tgetNode${nodeCase}s(filter: ${nodeCase}Input, options: Options): [${nodeCase}]\n`;
+        let nodeCase = replaceCleanseLabel(node.label);
+        let invalidNode = node.label !== nodeCase ? true : false;
+        nodeCase = toPascalCase(nodeCase);
+        
+        if (invalidNode) {
+            r += `\tgetNode${nodeCase}(filter: ${nodeCase}Input): ${nodeCase} @alias(property: "getNode${toPascalCase(node.label)}")\n`;
```

Comment:
I don't think these aliases are used either. When I comment out the logic to add this alias the query still works:
![Screenshot 2024-11-13 at 12 14 03 PM](https://github.com/user-attachments/assets/ebb652ad-8694-4bd8-9afb-3c13f4486a67)